### PR TITLE
Move CI to github action

### PR DESCRIPTION
**Pull request by w13915984028**
issue: https://github.com/harvester/harvester/issues/5327

### CODE REVIEW

**Comment by w13915984028** [comment]
@Vicente-Cheng I copied the changes from your csi driver PR, please help reviewing, thanks.

**Comment by Vicente-Cheng** [comment]
Hi @w13915984028 
The GHA path should be located `.github/workflows/xxx` instead `github/workflows/xxx`.

Please help to update again. Thanks!

File: scripts/build [hunk 1]
```diff
@@ -7,4 +7,9 @@ cd $(dirname $0)/..
 
 mkdir -p bin
 [ "$(uname)" != "Darwin" ] && LINKFLAGS="-extldflags -static -s"
-CGO_ENABLED=0 go build -ldflags "-X main.VERSION=$VERSION $LINKFLAGS" -o bin/harvester-cloud-provider
+
+#CGO_ENABLED=0 go build -ldflags "-X main.VERSION=$VERSION $LINKFLAGS" -o bin/harvester-cloud-provider
```
Comment (Vicente-Cheng) [inline scripts/build:11]: We could remove this line because we already have the following build command.

**Review by Vicente-Cheng — APPROVED** [review]
juat a nit, others LGTM. Thanks!

**Comment by bk201** [comment]
@w13915984028 We also need to create an EIO ticket to migrate secrets, for example: https://github.com/rancherlabs/eio/issues/2343

**Comment by w13915984028** [comment]
@bk201 The EIO ticket is on the way. thanks.

**Review by bk201 — APPROVED** [review]

**Pull request merged**